src/views/template/customerorder.html

```diff
@@ -1,6 +1,7 @@
-<div class="panel panel-default">
+<div class="panel panel-default" id="customer-{{ id }}">
```

Comment:
You could do a `data-customer="{{ id }}"`.

And then `$.data("customer")` on the appropriate object to retrieve the id.